### ISSUE TITLE
Running the pretrained for Go2

### ISSUE THREAD

**Issue by ducmngx** [littlebearqqq/legged_gym_go2#1]
Hi all,

I'm trying to use the pretrained model for Go2 in my project and I am not very sure what the inputs of the model are? I tried to plug it in skrl/Isaac-Velocity-Flat-Unitree-Go2-v0 in IsaacLab, the model is loaded, which means the model has an appropriate shape but Go2 doesn't move to track desired cmd_vel.

Can I have some advice on this issue? We are training a nav policy for go2 and really want to use Go2 pretrained model as our low level controller given the cmd_vel output from our main policy

**Comment by ducmngx** [comment]
I see. I am running my training with my trained model, which is ok but the robot is still crawling most of the time instead of walking.

**Comment by littlebearqqq** [comment]
> I see. I am running my training with my trained model, which is ok but the robot is still crawling most of the time instead of walking.

You can try to punish the hip joint more at lower speeds and less at higher speeds.
    def _reward_hip_pos(self):
        #return torch.sum(torch.square(self.dof_pos[:, [0, 3, 6, 9]] - self.default_dof_pos[:, [0, 3, 6, 9]]), dim=1)
        return (0.8-torch.abs(self.commands[:,1])) *  torch.sum(torch.square(self.dof_pos[:, [0, 3, 6, 9]] - self.default_dof_pos[:, [0, 3, 6, 9]]), dim=1)
